flowermd/base/simulation.py

```diff
@@ -67,6 +68,20 @@ def __init__(
         log_file_name="sim_data.txt",
         thermostat=HOOMDThermostats.MTTK,
     ):
+        if not isinstance(forcefield, Iterable) and not isinstance(
+            forcefield, str
+        ):
+            raise ValueError(
+                "forcefield must be a sequence of "
+                "hoomd.md.force.Force objects."
```

Comment:
If it can be a string, we should probably mention that here, too.

Reply:
It can't be a string

Reply:
I can't be a string, the 2nd conditional is because strings are iterable.  

```
>>> isinstance("gaff", Iterable)
True
```